spec/helpers/federal_register_documents_helper_spec.rb

```diff
@@ -61,18 +61,17 @@
   end
 
   describe '#federal_register_document_comment_period' do
-    context 'when federal_register_document_info is called' do
-      let(:result) do
-        "A <span>Notice</span> by the <span>Internal " \
-        "Revenue Service</span>, the <span>International " \
-        "Trade Administration</span> and the <span>National " \
-        "Oceanic and Atmospheric Administration</span> " \
-        "posted on <span>June 9, 2014</span>."
-      end
-
-      specify { expect(helper.federal_register_document_info(document)).to eq result }
+    let(:document) do
```

Comment:
This `let(:document)` block can be deleted, because this essentially duplicates the doc in line 4.

Reply:
Ack! that must have snuck in when I did git pull from release branch.